fets/model/mlcube/mlcube.yaml

```diff
@@ -24,4 +32,6 @@ tasks:
   # Run implementation
     parameters:
       inputs: {data_path: data/, parameters_file: {type: file, default: parameters.yaml}}
+      # TODO Is it possible to include optional inputs here (like alternative path to checkpoints)?
+      # inputs: {data_path: data/, parameters_file: {type: file, default: null}, checkpoint_path: {type: file, default: null}}
```

Comment:
Sure, the example you provided is good, we'll need to validate it as the string "null" though.

Reply:
Just replacing null -> "null" in the commented line doesn't work for me. MLCube still passes `mlcube_io1//.../fets/model/mlcube/workspace/null` to the mlcube.py. Can you elaborate what you meant?

Reply:
Sorry I was mistaken, we would need to validate if the defined optional input (null as a file path) exists, by default the validation will return false which means that the logic will use the default checkpoint and parameters file, but if the filesexist it will override the default values.

Reply:
Ok, that works, although I have to say that I would have liked passing `null` (not as string) better 😉